src/components/Card.js

```diff
@@ -115,7 +115,8 @@ export const Card = () => {
             </label>
           </div>
         </div>
-
+       <br />
```

Comment:
I'd prefer to add a `marginTop` on the `<hr>` element, instead of using a line break.